src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
+        if isinstance(batch, EnsembleSeries):
+            if batch.name == self._id_col:
+                batch = batch.rename("result")
+            batch = EnsembleFrame.from_dask_dataframe(batch.to_frame())
+            if len(on) > 1:
+                batch = batch.reset_index()
+                # Need to overwrite the meta manually as the multiindex will be
+                # interpretted by dask as a single "index" column
+                batch._meta = TapeFrame(columns=on + ["result"])
```

Comment:
1. Do we need dtypes here?
2. What if previously, on line 1165 `batch.name != self._id_col`? Would we have "result" column in that case?

Reply:
1. I'm not 100% sure, Dask's dtype inference seems much more robust than it's column label inference, so I haven't encountered any issues with leaving dtypes out. But it's possible there are some failure states I'm not accounting for.
2. We would not, and that's a good point as it might cause problems downstream in that case since we reference "result", maybe switching from result to batch.name throughout is more appropriate here

Reply:
Addressed 2. in latest commit

Reply:
Thank you